### ISSUE TITLE
Update instructions incorrect

### ISSUE THREAD

**Issue by Smileyone** [bpaauwe/udi-aeris-poly#12]
The update instructions say to go to Polyglot store and click on the update button, there is no such button, just an install button.
Polyglot update instructions say to stop and start the node, this doesn't work either. 

Also, there is no Polyglot dashboard Control->Restart

Thank you,

**Comment by CompKingCanada** [comment]
Top menu should display as: 
Dashboard / NodeServers /Settings / System / Log /Logout

Choose "System -> Restart Polygot"

Hope this fixes it!

**Comment by bpaauwe** [comment]
> The update instructions say to go to Polyglot store and click on the update button, there is no such button, just an install button.
> Polyglot update instructions say to stop and start the node, this doesn't work either.
> 
> Also, there is no Polyglot dashboard Control->Restart
> 
> Thank you,

You're right. The instructions were written for Polyglot, not Polyglot cloud.  Polyglot cloud always uses the most recent version of the node server when it's started.  However, Polyglot cloud also has a bug where it doesn't update the version number on the dashboard so you never know what version you are really running.   This behavior effects all node servers.

**Comment by Smileyone** [comment]
Thank you for the quick response.
